services/server/src/handlers/chat.ts

```diff
@@ -66,6 +69,12 @@ async function createRoom(req: Request, res: Response, next: NextFunction) {
   if (room) return next(exists.room());
 
   const roomId = await rooms.create(members);
+  if (message)
+    messages.create({
+      roomId,
+      userId: currentUser.id,
+      text: message,
+    });
```

Comment:
Creating a room and adding the first room message should be done in one transaction.